clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1636,14 +1637,11 @@ public void setInAppNotificationListener(InAppNotificationListener inAppNotifica
         coreState.getCallbackManager().setInAppNotificationListener(inAppNotificationListener);
     }
 
-    /**
-     * Returns the PushPermissionNotificationResponseListener object
-     *
-     * @return An {@link PushPermissionResponseListener} object
-     */
-    @SuppressWarnings({"unused", "WeakerAccess"})
-    public PushPermissionResponseListener getPushPermissionNotificationResponseListener() {
-        return coreState.getCallbackManager().getPushPermissionResponseListener();
+    @SuppressWarnings({"unused"})
+    public void unregisterPushPermissionNotificationResponseListener(PushPermissionResponseListener
```

Comment:
@shivamsharma2710 please add javadoc for the public methods

Reply:
This is done.